modules/virt-linux-bridge-nad-port-isolation.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-linux-bridge-nad-port-isolation.adoc_{context}"]
+= Enabling port isolation for a Linux bridge NAD
+
+You can enable port isolation for a Linux bridge network attachment definition (NAD) so that virtual machines (VMs) or pods that run on the same virtual LAN (VLAN) can operate in isolation from one another. The Linux bridge NAD creates a virtual bridge, or _virtual switch_, between network interfaces and the physical network. You can then configure port numbers on the virtual bridge so that only specific traffic can reach each port. 
+
+Isolating ports in this way can provide enhanced security, efficient resource use, and performance improvements for VM workloads that run on the same node. 
+
+.Prerequisites
+
+* For VMs, you configured either a static or dynamic IP address for each VM. See "Configuring IP addresses for virtual machines".
+* You created Linux bridge NAD by using either the web console or the command-line interface. 
+* You ensured that `spec.config.hairpinMode` in the Linux bridge NAD was not set to `true`.
+
+ts can be done statically inside the guest, or by a DHCP server in the physical network (the bridge is connected to)
+
+.Procedure
+
+. Edit the Linux bridge NAD by setting `portIsolation` to `true`:
++
+[source,yaml]
+----
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: bridge-network 
+  annotations:
+    k8s.v1.cni.cncf.io/resourceName: bridge.network.kubevirt.io/br1 
+spec:
+  config: |
+    {
+      "cniVersion": "0.3.1",
+      "name": "bridge-network", <1>
+      "type": "bridge", <2>
+      "bridge": "br1", <3>
+      "preserveDefaultVlan": false,
+      "vlan": 100,
+      "disableContainerInterface": "true", <4>
+      "portIsolation": "true" <5>
+    }
+# ...
+----
+<1> The name for the configuration. The name must match the the value in the `metadata.name` of the NAD.
+<2> The actual name of the Container Network Interface (CNI) plugin that provides the network for this network attachment definition. Do not change this field unless you want to use a different CNI.
+<3> The name of the Linux bridge that is configured on the node. The name should match the interface bridge name defined in the NodeNetworkConfigurationPolicy manifest.
+<4> Optional parameter that when set to `true` prevents the creation of network interfaces for each VM or pod in your cluster. A centralized data plane handles all container communications. If set to `false`, you cannot use IP address management (IPAM).
```

Comment:
> Optional parameter that when set to `true` prevents the creation of network interfaces for each VM or pod in your cluster.

When `disableContainerInterface` refers to the veth peer instead the container, actually the interface will created but in disabled state (state DOWN).

> A centralized data plane handles all container communications

For me its confusing and I am not sure what it means, I think this one should be omitted.

> If set to `false`, you cannot use IP address management (IPAM).

It should refer to the CNI  IPAM option that is sort of CNI plugin, aka IPAM CNI plugin.
I think it should mention it refer to plugin: ```you cannot use IP address management (IPAM) plugin.```

Reply:
:heavy_check_mark: